app/assets/javascripts/admin/tagify.js

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Question for me really rather than requesting a change.  
Why do we use javascript AND coffeescript in the same project ?  Seems confusing